easybuild/tools/job/gc3pie.py

```diff
@@ -197,7 +197,9 @@ def complete(self):
         self._engine.add(self.jobs)
 
         # in case you want to select a specific resource, call
-        self._engine.select_resource(build_option('job_target_resource'))
+        target_resource = build_option('job_target_resource')
+        if target_resource:
+            self._engine.select_resource(target_resource)
```

Comment:
Ok, I'll check the return value. But that's not the cause of the failing tests, since no target resource is specified there (and only one is available).

I need to change that to make the test robust, though...